dockers/docker-orchagent/start.sh

```diff
@@ -25,32 +25,28 @@ trap clean_up SIGTERM SIGKILL
 
 . /host/machine.conf
 
+HWSKU=`sonic-cfggen -m /etc/sonic/minigraph.xml -v minigraph_hwsku`
+
 MAC_ADDRESS=`ip link show eth0 | grep ether | awk '{print $2}'`
 
 ORCHAGENT_ARGS=""
 
-PORTSYNCD_ARGS=""
+PORTSYNCD_ARGS="-p /usr/share/sonic/$HWSKU/port_config.ini"
 
 SWSSCONFIG_ARGS="00-copp.config.json "
 
-if [ "$onie_platform" == "x86_64-dell_s6000_s1220-r0" ]; then
+if [ "$HWSKU" == "Force10-S6000" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
-    PORTSYNCD_ARGS+="-p /etc/ssw/Force10-S6000/port_config.ini"
     SWSSCONFIG_ARGS+="td2.32ports.qos.1.json td2.32ports.qos.2.json td2.32ports.qos.3.json td2.32ports.qos.4.json td2.32ports.qos.5.json td2.32ports.qos.6.json "
     SWSSCONFIG_ARGS+="td2.32ports.buffers.1.json td2.32ports.buffers.2.json td2.32ports.buffers.3.json "
-elif [ "$onie_platform" == "x86_64-dell_s6100_c2538-r0" ]; then
+elif [ "$HWSKU" == "Force10-S6100" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
-    PORTSYNCD_ARGS+="-p /etc/ssw/Force10-S6100/port_config.ini"
-elif [ "$aboot_platform" == "x86_64-arista_7050_qx32" ]; then
+elif [ "$HWSKU" == "Arista-7050-QX32" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
-    PORTSYNCD_ARGS+="-p /etc/ssw/Arista-7050-QX32/port_config.ini"
     SWSSCONFIG_ARGS+="td2.32ports.qos.1.json td2.32ports.qos.2.json td2.32ports.qos.3.json td2.32ports.qos.4.json td2.32ports.qos.5.json td2.32ports.qos.6.json "
     SWSSCONFIG_ARGS+="td2.32ports.buffers.1.json td2.32ports.buffers.2.json td2.32ports.buffers.3.json "
-elif [ "$onie_platform" == "x86_64-mlnx_x86-r5.0.1400" ] || [ "$onie_platform" == "x86_64-mlnx_msn2700-r0" ]; then
```

Comment:
Because nothing special is needed to do with it. port_config is done in line 34.

---
In reply to: [99408908](https://github.com/Azure/sonic-buildimage/pull/238#discussion_r99408908) [](ancestors = 99408908)